native/python/Makefile

```diff
@@ -24,7 +24,7 @@ PIP = $(WORK_DIR)/install/usr/local/bin/pip
 .PHONY: python_native_post_install
 python_native_post_install: $(WORK_DIR)/python-native.mk
 	@$(MSG) Installing setuptools, pip and cffi
-	@$(RUN) wget https://bootstrap.pypa.io/get-pip.py -O - | $(PYTHON)
+	@$(RUN) wget https://bootstrap.pypa.io/2.7/get-pip.py -O - | $(PYTHON)
```

Comment:
please do not fix this here, as it is not related to cross/numpy
_and I am already working on #4405_

BTW: you do not want to trigger the build of all python2 packages with the github build action.

Reply:
... You're right.
I will remove the commit !

Reply:
To be honest I tend to have some kind of obsession with "green" flags ;)

Et least we'll be able to see right there if it fixes the issue...

Reply:
I moved the change into a separate "branch" here in case it actually works: https://github.com/smaarn/spksrc/commit/53a0e18f69a1a93348e07101a4bd5d9e74753bf1